__tests__/app.test.js

```diff
@@ -119,3 +119,32 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+// 5. GET /api/articles/:article_id/comments
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("1. GET: 200 returns an array of comments for the given article_id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length).toBe(11);
+        response.body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
+        });
+      });
+  });
+  test("2. GET: 200 comments should be sorted by date in ascending order", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeSortedBy("created_at", { ascending: true });
+      });
+  });
+});
```

Comment:
Great start here Merin but missing error handling - please add the following tests and resubmit
- 400: what if an invalid `article_id` is given
- 404: what if a valid but non existent `article_id` is given
- 200: should return an empty array if an an article with no comments is requested